configs/MSI/Z170A-Gaming-Pro.conf

```diff
@@ -0,0 +1,42 @@
+chip "nct6793-*"
+
+    label in0 "Vcore"
+    
+    label in2 "AVSB"
+    set in2_min  3.3 * 0.90
+    set in2_max  3.3 * 1.10
+    
+    label in3 "3VCC"
+    set in3_min  3.3 * 0.90
+    set in3_max  3.3 * 1.10
+    
+    label in7 "3VSB"
+    set in7_min  3.3 * 0.90
+    set in7_max  3.3 * 1.10
+    
+    label in8 "Vbat"
+    set in8_min  3.0 * 0.90
+    set in8_max  3.0 * 1.10
+    
+    label in9 "Vtt"
+    set in9_min  1.0 * 0.90
+    set in9_max  1.0 * 1.10
+
+    set in1_max 2.048
+    set in4_max 2.048
+    set in5_max 2.048
+    set in6_max 2.048
+    set in10_max 2.048
+    set in11_max 2.048
+    set in12_max 2.048
+    set in13_max 2.048
+    set in14_max 2.048
```

Comment:
Do I understand it correctly that you don't know what these inputs are? If that's the case, I don't see the value of setting these limits. Can you explain?

Reply:
I don't know what those inputs are, but according to the spec sheet for NCT6793, they can have a maximum value of 2.048 V. Hence those max values.

Reply:
Well those are just the maximum values that the chip can detect. Are those somehow useful to the user? Do they relate to the maximum voltages that the chip will see on the corresponding pins? I'm not an expert, but I don't think so.

Reply:
They're not that useful, at best the user can see that they're not the 3V, 3.3V, 5V etc. inputs, but beyond that there's no use in setting those maximum values.

Reply:
In that case, I'd prefer to remove the settings. I think they might be confusing for users.

Reply:
Alternatively, you could add a comment explaining what the 2.048 values are. Whichever you prefer.